lua/modules/rapidxml.i

```diff
@@ -0,0 +1,22 @@
+%module RapidXML
```

Comment:
Vertall did this right?

Why even expose rapidxml? What purpose?

Reply:
Exposing RapidXML allows you to read the XML in LUA and do things based on it, so you can do things like add logic for a custom tag.

Reply:
Yeah but... I'd rather expose a Lua XML parser to Lua so it's easier to use if we're going to allow it, then maybe we can have a function to convert between RapidXML and whatever library format because that would probably be easier to deal with for developers. We only use RapidXML because FTL uses it internally and we had to hook some things that used it.

RapidXML is *really, really, clunky* plus not sure the security issues of that or not, needs more investigation if that's safe, it might be just fine.

Reply:
I'm pretty sure vert was able to confirm that exposing RapidXML doesn't let you do anything like read files outside of FTL's data, so I don't think security is an issue. I also don't really see the issue with using it over a native LUA parser, it's pretty smooth with just a couple helper functions. Plus I've already got the [Vertex mod](https://github.com/ChronoVortex/FTL-HS-Vertex) built on top of it and it would be a pain in the butt to change it :P

Reply:
Updates from vert:
- RapidXML is preferable to a LUA parser because it would be extremely difficult to set a LUA parser up to have parity with RapidXML. You don't want a situation where RapidXML is okay with something that the LUA parser isn't.
- Not certain about security, may need some additional testing.

Reply:
Alright, I've been thinking about this and I don't believe exposing RapidXML has any security issues. The only potential security concern I can think of, which has nothing to do with RapidXML, is exposing `Resources:LoadFile` allowing you to `../` into directories you shouldn't be able to read. It doesn't though, I've tested it and it only seems to let you work inside the `dat`.

Reply:
Can we split this part out into a separate PR? I think I want to try having us work on a layer that uses a Lua XML parser, yes maybe it has a method to convert to RapidXML. I think that would be a better approach?

Reply:
If you don't wanna expose RapidXML at all shouldn't I just remove it entirely instead of moving it to a different PR?

Reply:
Removed RapidXML